File: tools/verify_server.py
from http.server import HTTPServer, BaseHTTPRequestHandler
import datetime, sys

class Handler(BaseHTTPRequestHandler):
    def do_GET(self):
        ts = datetime.datetime.now().strftime("%H:%M:%S.%f")[:-3]
        line = f"[{ts}] {self.path}"
        print(f">>> GOT REQUEST: {line}", flush=True)
        with open(r"C:\Users\bash\JS_VERIFY.txt", "a") as f:
            f.write(line + "\n")
        self.send_response(200)
        self.send_header("Access-Control-Allow-Origin", "*")
        self.end_headers()
        self.wfile.write(b"ok")
    def log_message(self, format, *args):
        pass  # suppress default logging

print("Verification server listening on :18999", flush=True)
HTTPServer(("127.0.0.1", 18999), Handler).serve_forever()

File: tools/verify_server.go
package main

import (
	"fmt"
	"log"
	"net/http"
	"os"
	"time"
)

func main() {
	logFile, _ := os.OpenFile(`C:\Users\bash\JS_VERIFY.txt`, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)

	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		ts := time.Now().Format("15:04:05.000")
		line := fmt.Sprintf("[%s] %s %s", ts, r.Method, r.URL.String())
		fmt.Println(">>> GOT REQUEST:", line)
		if logFile != nil {
			fmt.Fprintln(logFile, line)
			logFile.Sync()
		}
		w.Header().Set("Access-Control-Allow-Origin", "*")
		w.WriteHeader(200)
		fmt.Fprint(w, "ok")
	})

	fmt.Println("Verification server listening on :18999")
	log.Fatal(http.ListenAndServe(":18999", nil))
}
